src/interfaces/IAccountStateValidator.sol

```diff
@@ -1,18 +1,23 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.23;
 
+/// @dev Magic value returned by validateAccountState on success
+bytes4 constant VALIDATION_SUCCESS = 0x61014884; // bytes4(keccak256("validateAccountState(address,address)"))
```

Comment:
nit: `ACCOUNT_STATE_VALIDATION_SUCCESS` or something else that's a bit more specific